dojo/widgets
# Upgrade to latest version of parade

**Type:** feature

The following has been addressed in the PR:

* [ ] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/framework/blob/master/STYLE.md)
* [ ] Unit tests are included in the PR
* [ ] For new widgets, an entry has been added to the `.dojorc`
* [ ] Any widget variant uses `theme.compose` like [this](https://github.com/dojo/widgets/issues/847)
* [ ] WidgetProperties are exported


**Description:**
Upgrade to latest parade:
* Update config
* Remove external css
* Add home markdown

Note:
Parade no longer does anything with `has` flags, so at the moment the `tests` menu option will show regardless of mode. In the future if parade gets passed no tests it will remove the menu item rather than relying on a flag.
